### DISCUSSION TITLE
Upgrade to 1.20 on Ubuntu fails on git ERROR

### DISCUSSION THREAD

**Discussion by emannersvip** [Chia-Network/chia-blockchain#7408] [Support]
My upgrade on all 4 of my Ubuntu 20.02 Chia machines fails with the same error below. The only machine I have running Chia 1.20 is when I removed the chia-blockchain directory and did a fresh install. Anyone else followed the upgrade docs without issue?

emanners@ThreeLeaf-Right:~/Crypto/Chia/chia-blockchain$ git status
On branch latest
Your branch is up to date with 'origin/latest'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
	modified:   chia-blockchain-gui (new commits, modified content)

no changes added to commit (use "git add" and/or "git commit -a")

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
